Zend/tests/constants/final_constants/final_const3.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+Private class constants cannot be final
+--FILE--
+<?php
+
+class Foo
+{
+    private final const A = "foo";
+}
+
+?>
+--EXPECTF--
+Fatal error: Private constant Foo::A cannot be final as it is never overridden in %s on line %d
```

Comment:
```suggestion
Fatal error: Private constant Foo::A cannot be final as it is not visible to other classes in %s on line %d
```
or something like that? I found the message a bit confusing.

Or possibly don't try to include reasoning at all:
```suggestion
Fatal error: Private constant Foo::A cannot be final in %s on line %d
```

Reply:
In my opinion the including the reason really helps understand why the error happens. Since the message format is copy pasted from the similar one for methods, I'll adapt that one as well.